src/connector/src/parser/mod.rs

```diff
@@ -118,7 +118,8 @@ impl<'a> MessageMeta<'a> {
                     CDC_TABLE_NAME_COLUMN_NAME,
                     "unexpected cdc meta column name"
                 );
-                Some(cdc_meta.full_table_name.as_str().into())
+                // For CDC sources, extract_table_name() strips the database prefix:
+                cdc_meta.extract_table_name()
```

Comment:
Is this change backward-compatible?
e.g. an existing connector's full_table_name might not contains database_name?